cognite/client/_api/simulators/simulator_models.py

```diff
@@ -0,0 +1,213 @@
+    ) -> SimulatorModelRevisionList:
+        """`Filter simulator model revisions <https://developer.cognite.com/api#tag/Simulator-Models/operation/filter_simulator_model_revisions_simulators_models_revisions_list_post>`_
+
+        List simulator model revisions
+
+        Args:
+            limit (int): The maximum number of simulator model revisions to return.
+            filter (SimulatorModelRevisionsFilter | dict[str, Any] | None): The filter to narrow down simulator model revisions.
+
+        Returns:
+            SimulatorModelRevisionList: List of simulator model revisions
+
+        Examples:
+
+            List simulator model revisions:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.models.revisions.list()
+
+        """
+        self._warning.warn()
+        return self._list(
+            method="POST",
+            limit=limit,
+            url_path="/simulators/models/revisions/list",
+            resource_cls=SimulatorModelRevision,
+            list_cls=SimulatorModelRevisionList,
+            filter=filter.dump()
+            if isinstance(filter, SimulatorModelRevisionsFilter)
+            else filter
+            if isinstance(filter, dict)
+            else None,
+        )
+
+    def retrieve(self, id: int | None = None, external_id: str | None = None) -> SimulatorModelRevision | None:
+        """`Retrieve simulator model revision <https://developer.cognite.com/api#tag/Simulator-Models/operation/retrieve_simulator_model_revisions_simulators_models_revisions_byids_post>`_
+
+        Retrieve multiple simulator model revisions by IDs or external IDs
+
+        Args:
+            id (int | None): The id of the simulator model revision.
+            external_id (str | None): The external id of the simulator model revision.
+
+        Returns:
+            SimulatorModelRevision | None: Requested simulator model revision
+
+        Examples:
+
+            Get simulator model revision by id:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.models.revisions.retrieve(id=123)
+
+            Get simulator model revision by external id:
+                >>> res = client.simulators.models.revisions.retrieve(external_id="abcdef")
+
+        """
+        identifiers = IdentifierSequence.load(ids=id, external_ids=external_id).as_singleton()
+        return self._retrieve_multiple(
+            list_cls=SimulatorModelRevisionList,
+            resource_cls=SimulatorModelRevision,
+            identifiers=identifiers,
+            resource_path="/simulators/models/revisions",
+        )
+
+    def retrieve_multiple(
+        self,
+        ids: Sequence[int] | None = None,
+        external_ids: SequenceNotStr[str] | None = None,
+        ignore_unknown_ids: bool = False,
```

Comment:
we don't support this param (other places as well), should be removed for now?

Reply:
If we don't supported in the API, we shouldn't have this here, I guess. Or is there some SDK magic that handles this case? @haakonvt